src/wazuh_modules/wm_github.c

```diff
@@ -349,6 +355,15 @@ STATIC void wm_github_execute_scan(wm_github *github_config, int initial_scan) {
                     wm_github_scan_failure_action(&github_config->fails, current->org_name, event_types[event_types_it], error_msg, github_config->queue_fd);
                 } else {
                     org_state_struc.last_log_time = new_scan_time;
+
+                    if (isDebug()) {
+                        memset(new_scan_time_str, '\0', 80);
```

Comment:
This code is not necessary, just the debug log.

Reply:
I agree